src/include/cpu/rpp_cpu_common.hpp

```diff
@@ -4995,10 +4995,10 @@ inline void compute_generic_bilinear_srclocs_and_interpolate(T *srcPtrChannel, R
 
     for (int c = 0; c < srcDescPtr->c; c++)
     {
-        dst[c] = (T)std::nearbyintf(((*(srcPtrChannel + srcLoc[0]) * bilinearCoeffs[0]) +        // TopRow R01 Pixel * coeff0
+        dst[c] = (T)((*(srcPtrChannel + srcLoc[0]) * bilinearCoeffs[0]) +       // TopRow R01 Pixel * coeff0
```

Comment:
Is this change necessary?
Would suggest to dump the old outputs and the current outputs with this change and see if those are matching

Reply:
yep checked all QA

Reply:
Instead of checking QA
Would suggest to dump the outputs with and without this change

Since we have 1 pixel threshold for QA. Wanted to see if there is any difference made

Reply:
We don't see any difference in outputs with this change, Previously we had added this to match HIP and HOST outputs right. Not very sure how come we don't see any effect of this change. Anyways I will revert my change.